src/engine/Settings.cs

```diff
@@ -538,6 +543,9 @@ public void ApplyAll(bool delayedApply = false)
     public void ApplyGraphicsSettings()
     {
         GUICommon.Instance.GetTree().Root.GetViewport().Msaa = MSAAResolution;
+
+        // Values less than 0 are undefined behaviour
+        Engine.TargetFps = MaxFramesPerSecond >= 0 ? MaxFramesPerSecond : 0;
```

Comment:
Is this related to the current value that is set in the project settings `settings/fps/force_fps` or should that be removed from the settings to not conflict with this?

Reply:
they are related, yes. As far as I can tell, the project's `force_fps` setting seems to set `Engine.TargetFps` at startup, so it should be safe to remove it now.

Reply:
If it overrides it anyway, I think it could be fine to keep. That way there is *some* max FPS limit set before our game code has a chance to run (just in case there can be problems).